src/index.ts

```diff
@@ -54,20 +54,30 @@ export function load(app: Application) {
 		vUtils.handleAssets(targetPath);
 		vUtils.handleJeckyll(rootPath, targetPath);
 
-		const directories = vUtils.getPackageDirectories(rootPath);
-		const semVers = vUtils.getSemVers(directories);
+		const metadata = vUtils.refreshMetadata(
+			vUtils.loadMetadata(rootPath),
+			rootPath,
+			vOptions.stable,
+			vOptions.dev
+		);
 
-		vUtils.makeStableLink(rootPath, semVers, vOptions.stable);
-		vUtils.makeDevLink(rootPath, vOptions.dev);
-		vUtils.makeMinorVersionLinks(semVers, rootPath);
+		vUtils.makeAliasLink(
+			'stable',
+			rootPath,
+			metadata.stable ?? metadata.dev
+		);
+		vUtils.makeAliasLink('dev', rootPath, metadata.dev ?? metadata.stable);
+		vUtils.makeMinorVersionLinks(metadata.versions, rootPath);
 
-		const jsVersionKeys = vUtils.makeJsKeys(semVers);
+		const jsVersionKeys = vUtils.makeJsKeys(metadata);
 		fs.writeFileSync(path.join(rootPath, 'versions.js'), jsVersionKeys);
 
 		fs.writeFileSync(
 			path.join(rootPath, 'index.html'),
 			'<meta http-equiv="refresh" content="0; url=stable"/>'
```

Comment:
We need to handle the possibility that "stable" may not exist yet for the url.

Reply:
Both symlinks always exist, they just aren't always listed in the drop-down.

Reply:
To elaborate, I made the decision that we should *always* have the symlinks available, because otherwise there will be URLs listed in READMEs etc. that sometimes exist and sometimes don't, and that's not a nice user experience. So keeping the symlinks always active is a must IMO. It's just sometimes they're not listed in the dropdown. In that event, the stable symlink just points to the same place that dev does, and vice versa.

Reply:
Still, I suppose I can make the root have a conditional redirect. Not too hard.

Reply:
Resolved by [97f3e77](https://github.com/citkane/typedoc-plugin-versions/pull/10/commits/97f3e77e56b9276e66e6488cb8e816802e95c281)